.travis.yml

```diff
@@ -28,13 +28,10 @@ before_cache:
   # The stats cache contains timestamped reports unused by CI but that
   # thrash the cache.  Skip caching these.
   - rm -rf ${HOME}/.cache/pants/stats
-  # While the bin directory and rust toolchains are relatively large, they're also very quick to
-  # restore/install: prune them to keep the total cache size down.
+  # While the bin directory is relatively large, it's also very quick to restore/install;
+  # so prune it to keep the total cache size down.
   #   see https://docs.travis-ci.com/user/caching/#Things-not-to-cache
-  # NB: We do _not_ prune the cargo cache, since that holds compiled tools and outputs, and
-  # individually resolved artifacts.
   - rm -rf ${HOME}/.cache/pants/bin
-  - rm -rf ${HOME}/.cache/pants/rust/rustup
```

Comment:
This directory is basically just `wget`'d by rustup; does caching it really give a noticeable improvement?

Reply:
I'm not sure - I did not isolate the two halves and test. I just setup the same caching Travis sets up for `cache: cargo` as noted in the associated issue. I'll run some experiments.

Reply:
Yea, would be good to isolate the two changes... caching the `target` directory makes a lot of sense, but this one maybe less so.

Reply:
Restored.